Automattic/woocommerce-payments
# Relocated client/utils.js into client/utils/index.js, so it was possible to use 'import {} from "utils"

Fixes #675

#### Changes proposed in this Pull Request

* Relocated client/utils.js into client/utils/index.js, as proposed in the original issue
* Migrated codebase to use `import { ... } from 'utils'`

See also https://github.com/Automattic/woocommerce-payments/issues/675#issuecomment-702599290, I would love to hear feedback on the findings as well =)

#### Testing instructions

* run `npm run test` to ensure everything works as expected

-------------------
